third_party/terraform/resources/resource_iam_binding.go.erb

```diff
@@ -141,6 +199,36 @@ func iamBindingImport(resourceIdParser resourceIdParserFunc) schema.StateFunc {
 		// Set the ID again so that the ID matches the ID it would have if it had been created via TF.
 		// Use the current ID in case it changed in the resourceIdParserFunc.
 		d.SetId(d.Id() + "/" + role)
+
+<% unless version == 'ga' -%>
+		// Read the upstream policy so we can set the full condition.
```

Comment:
We need to do a read of `condition` because we allow users to import by title but use the full contents of the condition as the `conditionKey` value, right?

Reply:
Yup, I expanded on the comment here to make that a bit more explicit.